install.ps1

```diff
@@ -0,0 +1,80 @@
+# Installs KoreBuild into a repo, overwriting all previous build scripts
+# This script is downloaded and executed without any of the corresponding scripts in this repo, so don't assume anything in this repo is available!
+
+param(
+    [string]$KoreBuildRepo,
+    [string]$KoreBuildBranch = "anurse/msbuild") # TEMPORARY until this is merged to dev
+
+# Prerequisites
+if(!(Get-Command git -ErrorAction SilentlyContinue)) {
+    throw "KoreBuild requires Git"
+}
+if(!(Test-Path "$(Get-Location)\.git")) {
+    throw "This install script should be run from the root of a Git repo"
```

Comment:
Then you can't **install** KoreBuild into it. If you download a repo that already has KoreBuild this script isn't ever used.